### PR TITLE
[zh-cn] sync daemonset.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/workloads/controllers/daemonset.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -222,13 +222,26 @@ the existing Pods based on the
 [priority](/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority)
 of the new Pod.
 -->
-DaemonSet 确保所有符合条件的节点都运行该 Pod 的一个副本。
+DaemonSet 被用来确保所有符合条件的节点都运行该 Pod 的一个副本。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/daemonset.md:225]:
```suggestion
DaemonSet 可用于确保所有符合条件的节点都运行该 Pod 的一个副本。
```

File: content/zh-cn/docs/concepts/workloads/controllers/daemonset.md [hunk 2]
```diff
@@ -222,13 +222,26 @@ the existing Pods based on the
...
 （驱逐）某些现存的 Pod。
 
+{{< note >}}
+<!--
+If it's important that the DaemonSet pod run on each node, it's often desirable
+to set the `.spec.template.spec.priorityClassName` of the DaemonSet to a
+[PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass)
+with a higher priority to ensure that this eviction occurs.
+-->
+当 DaemonSet 中的 Pod 在每个节点上运行非常重要时，通常需要将 DaemonSet
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/daemonset.md:239]:
```suggestion
当 DaemonSet 中的 Pod 必须运行在每个节点上时，通常需要将 DaemonSet
```
important 直翻不太顺口

File: content/zh-cn/docs/concepts/workloads/controllers/daemonset.md [hunk 3]
```diff
@@ -222,13 +222,26 @@ the existing Pods based on the
...
+的 `.spec.template.spec.priorityClassName` 设置为具有更高优先级的
+[PriorityClass](/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass)，
+以确保可以完成抢占（驱逐）。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/daemonset.md:242]:
```suggestion
以确保可以完成驱逐。
```
从字面看，抢占是 Preemption。technical 方面需要确认

**Review by ydFu — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**